values.yaml

```diff
@@ -282,9 +282,13 @@ minio:
     enabled: true
     name: "spinnaker"
   nodeSelector: {}
-# Uncomment if you don't want to create a PVC for minio
-#  persistence:
-#    enabled: false
+
+  # Minio persistent storage settings
+  persistence:
+    size: 30Gi
```

Comment:
The default I thought was 10G. So, here we are increasing it to 30G. But the PR title says reduce. 

So, what is the story here? And what is the issue faced that triggered this change?

Reply:
In the minio chart version (8.0.9) that is used a dependency, default storage size is 500Gi, which I believe is too huge. So, wanted to reduce it some lower value.